src/views/GameView.vue

```diff
@@ -113,61 +113,68 @@
+              <template v-if="!resolvingSeven">
+                <v-card-actions class="c-deck-count">({{ deckLength }})</v-card-actions>
+                <h1 v-if="deckLength === 0" id="empty-deck-text">PASS</h1>
+              </template>
 
-            <template v-if="resolvingSeven">
-              <p class="mt-2">Play from Deck</p>
-              <div class="d-flex">
-                <game-card
-                  v-if="topCard"
-                  :suit="topCard.suit"
-                  :rank="topCard.rank"
-                  :data-top-card="`${topCard.rank}-${topCard.suit}`"
-                  :is-selected="topCardIsSelected"
-                  class="mb-4 resolving-seven-card"
-                  @click="selectTopCard"
-                />
-                <game-card
-                  v-if="secondCard"
-                  :suit="secondCard.suit"
-                  :rank="secondCard.rank"
-                  :data-second-card="`${secondCard.rank}-${secondCard.suit}`"
-                  :is-selected="secondCardIsSelected"
-                  class="mb-4 resolving-seven-card"
-                  @click="selectSecondCard"
-                />
-              </div>
-            </template>
-          </v-card>
-          <scrap-dialog :scrap="scrap">
-            <template #activator>
-              <div id="scrap" class="d-flex flex-column align-center">
-                <h3>Scrap</h3>
-                <span>({{ scrap.length }})</span>
-                <v-btn variant="outlined" color="primary" class="mt-4"> View </v-btn>
-              </div>
-            </template>
-          </scrap-dialog>
+              <template v-if="resolvingSeven">
+                <p class="mt-2">Play from Deck</p>
+                <div class="d-flex">
+                  <game-card
+                    v-if="topCard"
+                    :suit="topCard.suit"
+                    :rank="topCard.rank"
+                    :data-top-card="`${topCard.rank}-${topCard.suit}`"
+                    :is-selected="topCardIsSelected"
+                    class="mb-4 resolving-seven-card"
+                    @click="selectTopCard"
+                  />
+                  <game-card
+                    v-if="secondCard"
+                    :suit="secondCard.suit"
+                    :rank="secondCard.rank"
+                    :data-second-card="`${secondCard.rank}-${secondCard.suit}`"
+                    :is-selected="secondCardIsSelected"
+                    class="mb-4 resolving-seven-card"
+                    @click="selectSecondCard"
+                  />
+                </div>
+              </template>
+            </v-card>
+            <point-counter
+              id="opponent-point-counter"
+              class="point-container point-counter"
+              :king-count="opponentKingCount"
+              :points-to-win="opponentPointsToWin"
+              :is-player="false"
+            />
```

Comment:
@jerryh-jr Here's the layout change - since we want the point counters aligned with the deck/scrap piles, I moved the point counter components to the `deck-container` grid area. Beyond that, I added each point counter to a div with it's respective deck element, with a class called `deck-outer` that handles the alignment